docs/src/pages/components/link/examples/LinkGlobalStylingExample.tsx

```diff
@@ -0,0 +1,6 @@
+import { Link, useTheme } from '@aws-amplify/ui-react';
+
+export const LinkGlobalStylingExample = () => {
+  const { tokens } = useTheme();
+  return <Link color={tokens.colors.red[80]}>Link Global Styling</Link>;
```

Comment:
The example and the code don't match here, the example is global styling but here you are using local styling.  We don't want to override the global styling across the entire page but using a class name here and including the updated style in the corresponding css file would be a more accurate representation of what we are trying to demonstrate